docs/modules/ROOT/pages/how-tos/new_minor.adoc

```diff
@@ -0,0 +1,52 @@
+= Checklist to prepare for a new minor version of OpenShift 4
+:fn-core-components: pass:c,q[footnote:[Core Commodore components are those included by the distribution `openshift4` and its parent hierarchy levels.]]
+
+. Read and summarize release notes
++
+Read the release notes.
+Look for things that affect how we use OpenShift.
+Ensure each change which impacts us gets addressed.
++
+Write a summary and add it to xref:oc4:ROOT:references/release_notes.adoc[our release notes summary].
+Make note of changes that influence how the solution teams operate a cluster or how our users use their cluster.
+Keep only the three most recent versions and delete the older ones.
+
+. Conduct upgrade
++
+Use an existing test cluster and upgrade it to the new version.
++
+[NOTE]
+====
+The new minor might not show up in the list of available updates, even when setting the channel accordingly.
+If this is the case, set the cluster's update channel to `fast-<n-1>` and update to the latest patch first.
+This is, amongst others, necessary, when there is no GA release for the new minor available yet.
+The first GA version of a new minor usually is `4.<n>.3`.
+====
++
+[TIP]
+====
+Instead of using an existing cluster, you can also set up one with version `4.<n-1>` and then upgrade this one.
+====
+
+. Check compatibility of components
++
+Check if all core Commodore components{fn-core-components} are compatible with the new version.
```

Comment:
```suggestion
Check if all core Commodore components footnote:[Core Commodore components are those included by the distribution `openshift4` and its parent hierarchy levels.] are compatible with the new version.
```

There is no need for the complicated `{fn-core-components}` construct here...

Reply:
I kind of disliked the interleaved sentences when looking at the asciidoc. It does not matter when rendered.